pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go

```diff
@@ -397,7 +397,7 @@ func (dswp *desiredStateOfWorldPopulator) getPVSpec(
 	}
 
 	volumeGidValue := getPVVolumeGidAnnotationValue(pv)
-	return volume.NewSpecFromPersistentVolume(pv, pvcReadOnly), volumeGidValue, nil
+	return volume.NewSpecFromPersistentVolume(pv, pvcReadOnly, "default"), volumeGidValue, nil
```

Comment:
Modify this method to accept `podNamespace` and use it here instead of `"default"`.